dev/devicelab/lib/framework/framework.dart

```diff
@@ -69,7 +69,7 @@ class _TaskRunner {
     });
     registerExtension('ext.cocoonRunnerReady',
         (String method, Map<String, String> parameters) async {
-      return ServiceExtensionResponse.result('"ready"');
+      return ServiceExtensionResponse.result('{"result":"ready"}');
```

Comment:
The docs on this say that it has to be a valid "JSON object", which I assume means a `{}`. `package:vm_service` actually expects this to correspon to a `Map` object in Dart, whereas `vm_service_client` tolerated a String. A quoted string is valid JSON, but I guess not a valid JSON object.

/cc @bkonyi FYI.